org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -800,6 +800,54 @@ GithubOidcOpenChallengesDeploy:
       - !Ref ITSandboxAccount
     Region: us-east-1
 
+# Permissions to push an image to private registry
+# See: https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-push-iam.html
+GithubOidcNbConvertDeploy:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-nbconvert-deploy
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-nbconvert-deploy
+    MaxSessionDuration: 7200
+    PolicyDocument: >-
+      {
+          "Version": "2012-10-17",
+          "Statement": [
+              {
+                  "Effect": "Allow",
+                  "Action": [
+                      "ecr:CompleteLayerUpload",
+                      "ecr:UploadLayerPart",
+                      "ecr:InitiateLayerUpload",
+                      "ecr:BatchCheckLayerAvailability",
+                      "ecr:PutImage"
+                  ],
+                  "Resource": [
+                      "arn:aws:ecr:region:449435941126:repository/nbconvert",
+                      "arn:aws:ecr:region:449435941126:repository/dev-nbconvert",
+                      "arn:aws:ecr:region:325565585839:repository/prod-nbconvert"
```

Comment:
does this work?  when this is deployed to `SynapseDevAccount` then i think `arn:aws:ecr:region:325565585839:repository/prod-nbconvert` is not available because it's in `SynapseProdAccount`?

Reply:
I'm actually not sure, do you think there's a check either at deploy time or at run time for all the values? My thinking was there's not and even if I tried to push cross account (which technically is allowed by this policy) I would be blocked by not having the cross account permission set up.
Ideally, I would just have used wildcards but they're not permitted as I found out in a previous PR.
If this does not work, then I would have to split the policy to match the account.